paper/paper.md

```diff
@@ -0,0 +1,519 @@
+live-coding and hands-on practice.
+However, we couldn’t find a resource that exactly met our needs. Therefore, over the past
+three years, we have developed a curriculum and instructional format using
+Jupyter notebooks to effectively teach introductory Python for data science.
+This method, inspired by The Carpentries organization, uses bite-sized lessons
+followed by independent practice time to reinforce coding concepts, and
+culminates in a data science capstone project using real-world data.
+We believe our open curriculum is a valuable resource to the wider education
+community and hope that educators will use and improve our lessons, practice
+problems, and teaching best practices.
+Anyone can contribute to [our educational materials on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women bioinformatics trainees at the
+University of Michigan, we experience the gender gap in our field first-hand.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We also see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level; during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program was
+35.2%, and the average percentage of female applicants listing Bioinformatics as first or second
+choice in the Program in Biomedical Sciences, U-M's biomedical PhD umbrella
+program was 41%.
+
+Previous research on women's educational experiences in science, technology, engineering,
+and mathematics (STEM) have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+Additionally, women often experience stereotype threats that
+influence their math and science performance as well as their career goals [@hill_why_2010].
+The majority of our organization's founding graduate students (all women) began
+coding in our undergraduate careers or later.
+We wanted to provide a safe place for local high school women to
+develop confidence in themselves and their computational skills
+before college, and be exposed to successful women role models in STEM to counter negative
+stereotypes.
+
+Girls Who Code, a national organization whose mission
+is to close the gender gap in technology
+[@saujani_girls_2015], was founded in 2012.
+Because of our personal experiences and the paucity of women in our field, we
```

Comment:
Do you think we should cite this?

Reply:
Do you mean add a citation for "the paucity of women in our field"?

Reply:
Yes

Reply:
@bnwolford do you have a good reference for this?

Reply:
Could reference this: @nces_digest_2012 (we reference it in relation to computer and information science earlier)

```suggestion
Because of our personal experiences and the paucity of women in our field [@nces_digest_2012], we
```